RELEASE/scripts/autoscend.ash

```diff
@@ -2709,24 +2711,49 @@ boolean LX_getDigitalKey()
 	{
 		return false;
 	}
+	if(item_amount($item[Digital Key]) > 0)
+	{
+		if(have_effect($effect[Consumed By Fear]) > 0)
+		{
+			uneffect($effect[Consumed By Fear]);
+			council();
+		}
+		return false;
+	}
+	
+	//why is this here?
 	if (internalQuestStatus("questL12War") == 1 && get_property("sidequestArenaCompleted") != "none" && get_property("auto_powerLevelAdvCount").to_int() < 9)
```

Comment:
"If you don't need to flyer for the war, and you're still doing the war, finish the war before adventuring here"

this makes sense at least because of the Powerful Glove, but possibly for other reasons?

Reply:
This is one of those "we assume everyone is doing the frat side of the war" things.
The conditions are

- is the war started
- are you still flyering
-  have you attempted to powerlevel less than 9 times.

It's trying to stop you using the digital key acquisition to flyer until you've powerlevelled for 9 adventures. That seems silly to me, that last condition would be better as `my_level() == get_property("auto_powerLevelLastLevel").to_int()` (which should also be in a function somewhere so we can re-use it). I think Cheese was fond of stuff like this as it made the script look more intelligent that it actually is.

Reply:
thanks for the explanation on what it is doing. The powerful glove now has its own seperate block so is this still even needed? I am pretty sure this bit of code far predates the powerful glove too.

The thing is, it is not waiting to do this zone until you unlock the war. it is refusing to do this zone if you have flyers in hand (as fratboy) but did not power level yet... why? is it because power leveling gives more ML to flyer with? but this is still free ML when doing this zone so there is no reason to skip it.

It would have been more logical if it was "if you have not yet flyered, don't do this zone". a check which incidentally should be done later in the function (below the crafting and above the adventuring code)

As it is though, I am thinking this line should just be removed.

Reply:
removed on commit dda9a58